src/listeners/issue-embedder.ts

```diff
@@ -0,0 +1,25 @@
+import { Message } from 'discord.js';
+
+import { getIssueNumbers } from '../utilities/get-issue-numbers';
+import { issueEmbedGenerator } from '../utilities/issue-embed-generator';
+import { GithubResponse } from '../APIs/github-def';
+import { logger } from '../utilities/logger';
+import { getIssueData } from '../utilities/get-issue-data';
+
+export function issueEmbedder(message: Message): void {
+  const issues = getIssueNumbers(message.content);
+
+  if (issues.length > 0) {
+    issues.forEach(async (issueNumber) => {
```

Comment:
This setup has the problem of potentially returning issues out of order. Change to a `.map` + `Promise.all` so we can keep track of the order. 

Also I think maybe we should put a limit on how many can returned at the same time? Otherwise it would be easy to spam. (Maybe 3-5, or set as an environment variable)

Reply:
Personally I don't see a need to link more than one issue at a time.

Reply:
I've limited the auto linking to only 5 items by default and it's customizable by changing the env variables.